tutorials/axios/axios-multi-form-data.md

```diff
@@ -0,0 +1,17 @@
+To send multipart form data with axios, there is some preparation
```

Comment:
Sentence is too long and unnecessary. In the browser, there's no setup necessary. I'd just replace this paragraph with "To send multipart form data with Axios, you need to use the `FormData` class. Browsers have a built-in `FormData` class, but Node.js doesn't, so you need to use the `form-data` npm module."